react-app/package.json

```diff
@@ -22,76 +22,79 @@
+    "@romcal/calendar.bosnia-herzegovina": "3.0.0-dev.90",
+    "@romcal/calendar.brazil": "3.0.0-dev.90",
+    "@romcal/calendar.canada": "3.0.0-dev.90",
+    "@romcal/calendar.chile": "3.0.0-dev.90",
+    "@romcal/calendar.china": "3.0.0-dev.90",
+    "@romcal/calendar.costa-rica": "3.0.0-dev.90",
+    "@romcal/calendar.croatia": "3.0.0-dev.90",
+    "@romcal/calendar.czech-republic": "3.0.0-dev.90",
+    "@romcal/calendar.denmark": "3.0.0-dev.90",
+    "@romcal/calendar.england": "3.0.0-dev.90",
+    "@romcal/calendar.europe": "3.0.0-dev.90",
+    "@romcal/calendar.finland": "3.0.0-dev.90",
+    "@romcal/calendar.france": "3.0.0-dev.90",
+    "@romcal/calendar.france.coutances": "3.0.0-dev.90",
+    "@romcal/calendar.france.lyon": "3.0.0-dev.90",
+    "@romcal/calendar.france.paris": "3.0.0-dev.90",
+    "@romcal/calendar.france.saint-denis": "3.0.0-dev.90",
+    "@romcal/calendar.france.strasbourg": "3.0.0-dev.90",
+    "@romcal/calendar.general-roman": "3.0.0-dev.90",
+    "@romcal/calendar.germany": "3.0.0-dev.90",
+    "@romcal/calendar.greece": "3.0.0-dev.90",
+    "@romcal/calendar.guatemala": "3.0.0-dev.90",
+    "@romcal/calendar.hungary": "3.0.0-dev.90",
+    "@romcal/calendar.india": "3.0.0-dev.90",
+    "@romcal/calendar.ireland": "3.0.0-dev.90",
+    "@romcal/calendar.italy": "3.0.0-dev.90",
+    "@romcal/calendar.japan": "3.0.0-dev.90",
+    "@romcal/calendar.lebanon": "3.0.0-dev.90",
+    "@romcal/calendar.lithuania": "3.0.0-dev.90",
+    "@romcal/calendar.malta": "3.0.0-dev.90",
+    "@romcal/calendar.mexico": "3.0.0-dev.90",
+    "@romcal/calendar.netherlands": "3.0.0-dev.90",
+    "@romcal/calendar.new-zealand": "3.0.0-dev.90",
+    "@romcal/calendar.norway": "3.0.0-dev.90",
+    "@romcal/calendar.panama": "3.0.0-dev.90",
+    "@romcal/calendar.paraguay": "3.0.0-dev.90",
+    "@romcal/calendar.peru": "3.0.0-dev.90",
+    "@romcal/calendar.philippines": "3.0.0-dev.90",
+    "@romcal/calendar.poland": "3.0.0-dev.90",
+    "@romcal/calendar.portugal": "3.0.0-dev.90",
+    "@romcal/calendar.puerto-rico": "3.0.0-dev.90",
+    "@romcal/calendar.romania": "3.0.0-dev.90",
+    "@romcal/calendar.russia": "3.0.0-dev.90",
+    "@romcal/calendar.scotland": "3.0.0-dev.90",
+    "@romcal/calendar.slovakia": "3.0.0-dev.90",
+    "@romcal/calendar.slovenia": "3.0.0-dev.90",
+    "@romcal/calendar.spain": "3.0.0-dev.90",
+    "@romcal/calendar.sri-lanka": "3.0.0-dev.90",
+    "@romcal/calendar.sweden": "3.0.0-dev.90",
+    "@romcal/calendar.switzerland": "3.0.0-dev.90",
+    "@romcal/calendar.ukraine": "3.0.0-dev.90",
+    "@romcal/calendar.united-states": "3.0.0-dev.90",
+    "@romcal/calendar.uruguay": "3.0.0-dev.90",
+    "@romcal/calendar.venezuela": "3.0.0-dev.90",
+    "@romcal/calendar.vietnam": "3.0.0-dev.90",
+    "@romcal/calendar.wales": "3.0.0-dev.90",
     "@tanstack/react-query": "^5.56.2",
     "classnames": "^2.5.1",
     "date-fns": "^4.1.0",
     "flatted": "^3.3.1",
+    "i18next": "^23.15.2",
```

Comment:
This has been new since converting the Romcal codebase into a monorepo. I'm not sure if it's a bug or an improvement... Previously, i18next was bundled with Romcal's source code, but now it has to be added explicitly to use Romcal.